test/TestInternalGrains/PersistenceTestGrains.cs

```diff
@@ -290,6 +291,14 @@ public Task DoSomething()
         }
     }
 
+    public class ServiceIdGrain : Grain, IServiceIdGrain
+    {
+        public Task<Guid> GetServiceId()
+        {
+            return Task.FromResult(this.ServiceProvider.GetRequiredService<GlobalConfiguration>().ServiceId);
```

Comment:
Why not get this from the strongly typed option (instead of the legacy config) which is the authoritative source of truth?

Reply:
Will fix